doc/tutorials/content/compiling_pcl_docker.rst

```diff
@@ -0,0 +1,154 @@
+.. _compiling_pcl_docker:
+
+Compiling PCL from source on Docker
+===================================
+
+This tutorial explains how to build the Point Cloud Library **from source** on docker.
+
+.. note::
+
+   The walkthrough of the procedure was done in Windows 10 host machine where Ubuntu 20.04
+   was installed in virtual machine. The reason is that docker can be much easier installed
+   in linux OSs compared to Windows
```

Comment:
I find this a bit confusing (Windows 10 is the host for a Ubuntu 20.04 virtual machine, which in turn is the host for a Ubuntu 20.04 docker container?), and it does not seem relevant for the tutorial. I would only say that the tutorial assumes that the (docker) host is Ubuntu, that other operating systems may also be used as hosts (possibly with some modifications to the commands), and (if you really want) that the docker host can itself run inside a virtual machine.

Reply:
As you said, Windows 10 is the host for a Ubuntu 20.04 virtual machine, which in turn is the host for a Ubuntu 20.04 docker container. Anyway, i will update it.